projects/angular-showcase/src/app/app.module.ts

```diff
@@ -10,6 +10,7 @@ import { MonacoEditorModule } from 'ngx-monaco-editor';
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 import { IntroductionComponent } from './introduction/introduction.component';
+import { MapsModule } from './maps/maps.module';
```

Comment:
Remove this import, since it will be imported via lazy loading and this import would break that